torch/_subclasses/fake_tensor.py

```diff
@@ -1034,13 +1034,28 @@ def merge_devices(t):
 # memory should not significantly incraese.
 
 
+@contextlib.contextmanager
+def disable_fake_tensor_mode_tracing():
+    modes = torch.utils._python_dispatch._get_current_dispatch_mode_stack()
+    fake_tensor_modes = [m for m in modes if isinstance(m, FakeTensorMode)]
+    olds = [m.enable_tracing for m in fake_tensor_modes]
+    for fake_mode in fake_tensor_modes:
+        fake_mode.enable_tracing = False
+    try:
+        yield
+    finally:
+        for fake_mode, old in zip(fake_tensor_modes, olds):
+            fake_mode.enable_tracing = old
+
+
 class FakeTensorMode(TorchDispatchMode):
     def __init__(
         self,
         *,
         allow_fallback_kernels=True,
         allow_non_fake_inputs=False,
         shape_env=None,
+        enable_tracing=True,
```

Comment:
This is very very questionable. You can't just skip FakeTensorMode, you may improperly attempt to access fake tensors data pointers and crash. It's not clear to me why you are doing this yet since I haven't read the rest of the PR.